be/src/common/config.h

```diff
@@ -486,7 +486,7 @@ CONF_Int32(load_process_max_memory_limit_percent, "50");         // 50%
 // soft limit which can trigger the memtable flush for the load channel who
 // consumes lagest memory size before we reach the hard limit. The soft limit
 // might avoid all load jobs hang at the same time.
-CONF_Int32(load_process_soft_mem_limit_percent, "50");
```

Comment:
Why change load_process_soft_mem_limit_percent from 50% to 80%?

Reply:
50% is too low, very easy to trigger frequent flush. and in our test, leaving 20% memory is enough.